slu/slu/constants/__init__.py

```diff
@@ -130,9 +132,16 @@
 CURRENT_INTENT = "current_intent"
 STATE = "state"
 EXPECTED_SLOTS = "expected_slots"
-
-LANG_TO_LOCALES = {"en": "en_IN", "hi": "hi_IN"}  # This should be set via config
-
+NLS_LABEL = "nls_label"
+PROMPT_NOISE_FILLER_TOKEN = "<pad>"
+LANG_TO_LOCALES = {                     
+    "en": "en_IN", 
+    "hi": "hi_IN"
+}                                        # This should be set via config
+PLATFORM_LEVEL_NOISE = {                     
+    "en": ["en_nls", "en_us"], 
+    "hi": ["hi_nls","hi_us"]
+}
```

Comment:
Read your comment on the other thread for this constant. It's a fair thing to do.. Let's add a description for why this is a list as value. (just whatever you wrote in that thread)